packages/utils/src/stringUtils.ts

```diff
@@ -1,9 +1,10 @@
 export const formaterFritekst = (value: string | number) =>
-  value
-    .toString()
+  value.toString()
     .replace(/\p{Dash_Punctuation}/gu, '-')
     .replace(/[\p{Space_Separator}\t]/gu, ' ')
     .replace(/[“”]/g, '"')
     .replace(/[‘’`]/g, "'")
+    .replace(/&/g, "&amp;")
     .replace(/</g, "&lt;")
     .replace(/>/g, "&gt;")
+    .replace(/\//g, "&quot;")
```

Comment:
Slik eg ser det går ein her frå å tillate at bruker skriver inn tekst med `&` og `/` i til å ikkje tillate det lenger. Og `/` blir konvertert til `"`. Er det eigentleg ønskeleg? Ville tru det kan gje ein forvirrande/dårleg brukeropplevelse.

Eit meir generelt spørsmål er kvifor ein meiner det er nødvendig. Det ser for meg ut til at det har vore tillatt fram til no.

Er det nokon stad kjent mulighet for at tekst blir satt inn som html i DOM?

Reply:
Det var en typo der hvor en endrer fra / til ". Har fjernet den. Kan forsåvidt også fjerne & her. 

Om det er kjent mulig så kan ikke eg gi deg noe fasit svar der. 

Intensjonen her er å støtte flere tegn slik at saksbehandler ikke alltid må fjerne disse i etterkant. Mange kopierer fra dokumenter andre plasser (doc/pdf) som tar med seg en del ugyldige tegn den dag i dag. Så her forsøker vi å vaske dataen som sendes inn fremfor å be de selv finne og erstatte tegn. 

Her har vi strengere input validering i backend som f.eks. å ikke lov med < og >.

Reply:
Forstår, denne ser ok ut for min del no 👍